src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php

```diff
@@ -191,17 +191,6 @@ public function testAuthFromOptionsAndDsn()
         Connection::fromDsn('redis://password1@localhost/queue', ['auth' => 'password2'], $redis);
     }
 
-    public function testAuthAsUserInDsn()
-    {
-        $redis = $this->createMock(\Redis::class);
-
-        $redis->expects($this->exactly(1))->method('auth')
-            ->with('password')
-            ->willReturn(true);
-
-        Connection::fromDsn('redis://password:localhost/queue', [], $redis);
-    }
-
```

Comment:
This test was added in #42067 but it never passed, the dsn is invalid (see https://github.com/symfony/symfony/runs/3097443193)